src/schema.ts

```diff
@@ -5,6 +5,7 @@ export const typeDefs = buildSchema(`
 type Query {
     hello: String
     user (id: Int): UserOutput!
+    users (limit: Int): [UserOutput]
```

Comment:
```suggestion
    users (limit: Int = 10): [UserOutput]
```

Se não me engano assim vc decalra um valor padrão para esse argumento no schema, se quier. Aí imagino que ele já venha preenchido com esse valor padrão no resolver caso não seja enviado. Mas vale testar, não cheguei a fazer com esse schema manual.

Reply:
Opa Alan, testei aqui e funcionou com essa declaração padrão no schema!